docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -395,4 +395,72 @@ This script generates a timestamped log file (e.g., `ac-log-2024-10-01-14-55.txt
 
 Ensure that the **Custom Script** step runs after the [**Export Build Artifacts**](/workflows/common-workflow-steps/export-build-artifacts) step to capture the full log.
 
-:::
\ No newline at end of file
+:::
+
+### How can I print the status of workflow steps with detailed information?
+
+If you want to track or share the status of your workflow or individual steps during a build, you can use the following environment variables:  
+
+- **`$AC_BUILD_STATUS`**: Displays the overall status of the workflow.
+- **`$AC_BUILD_STEP_STATUS`**: Displays detailed information about each executed step.  
+
+However, the output of `$AC_BUILD_STEP_STATUS` is in raw JSON format, which may not be easy to read directly. To make it more readable, you can use the following Ruby script to format and print the information in a user-friendly way:
```

Comment:
Env variable naming check (same here). In the sample script and screenshots from issue: `AC_BUILD_STEPS_STATUS`